usr.bin/sockstat/sockstat.c

```diff
@@ -1107,235 +1093,420 @@ sctp_path_state(int state)
 	}
 }
 
+static int
+format_unix_faddr(struct addr *faddr, char *buf, size_t bufsize) {
+	size_t pos = 0;
+	/* Remote peer we connect(2) to, if any. */
+	if (faddr->conn != 0) {
+		struct sock *p;
+		strlcpy(buf, "-> ", bufsize);
+		pos += 3;
```

Comment:
Better to rely on strlcpy returning the right value than to hard-code it.
```suggestion
		pos += strlcpy(buf, "-> ", bufsize);
```